proto/viam/app/data/v1/data.proto

```diff
@@ -179,6 +189,22 @@ message BinaryDataByIDsResponse {
   uint64 count = 2;
 }
 
+// BoundingBox represents a labeled bounding box on an image.
+// x and y values are normalized ratios between 0 and 1.
+message BoundingBox {
```

Comment:
do we need bson values for this one? Is it stored in mongo in this format?

Reply:
bson values are accounted for in `dbAnnotation`, which is our internal representation of the bounding box, so I believe no need to include it here